datasketch/storage.py

```diff
@@ -365,7 +365,7 @@ def remove_val(self, key, val):
                 self._redis.hdel(self._name, redis_key)
 
         def insert(self, key, *vals, **kwargs):
-            # Using buffer=True outside of an `insertion_session`
+            # Using buffer=True outside of an `session`
```

Comment:
This will break existing code that uses the stable API. Let's keep the changes localized within the `experimental` submodule for now.

Reply:
You mean because of **kwargs? Well, I thought about minimum difference with main code. This happens because redis storage do not support bulk operations (see comment below on your question) but mongo supports. So, to use it from one interface I added **kwargs. But I can't understand how I's going to breake existing code. Maybe you can be more specific.

Reply:
Yes. I think the comment in this function, which is a public API, may lead to misunderstanding.

I thought Redis does support [bulk operation](https://redis.io/topics/mass-insert). Anyway I think it is better to keep the interface, including existing kwargs, unchanged.